src/app/(courseList)/loading.tsx

```diff
@@ -0,0 +1,30 @@
+import CourseListHeader from '@components/CourseList/CourseListHeader'
+import LoadingItem from '@components/CourseList/Loading'
+
+import { getSessionUser } from '@services/tkoUserService'
+
+const Loading = async () => {
+  const { rights } = await getSessionUser()
+
+  return (
+    <main
+      role="alert"
+      aria-label="Loading courses..."
+      className="list-container divide-y pb-5"
+    >
+        <CourseListHeader showManage={rights.rename || rights.remove} />
+        <LoadingItem showManage={rights.rename || rights.remove} />
```

Comment:
To reduce repetetition, maybe a `<LoadingItems amount={10} showManage=... />` wrapper? If this is the only place where this is used then alright, but I assume there are others.

Reply:
In Next.js App folder structure, loading elements are only defined in `loading.tsx` files. I think that this would lead into more complexity if separated into wrapper. This particular loading wrapper would only be used in course and files lists loading pages.